### PR TITLE
[CGPROD-2701] nudge devs towards achievement variety

### PR DESCRIPTION
**Pull request by BeeGodwin**
- Updated `achievements.md` - remove redundant instruction on enabling achievements, clearer guidance on renaming / reordering achievements
- Updated `themes/default/achievements/config.json5` - explanatory comments, prefix strings CHANGE_THESE_STRINGS for clarity

### CODE REVIEW

File: docs/development/achievements.md [hunk 1]
```diff
@@ -1,22 +1,12 @@
 # Achievements System
 
-## How To Enable
-Achievements are enabled and configured per theme.
-To enable the achievements system add `"achievements": true` to the game block of the theme config `themes/#####/config/config.json5`. e.g:
```
Comment (BeeGodwin) [inline docs/development/achievements.md:5]: This appears to be outdated- at least, it's not in the demo 'game' but it still has achievements. 

Looks like achievements are 'always on' and we can turn them off by passing an empty array in the achievements config.json5  ...updated here, but tell me if I'm wrong 😂

Comment (FostUK) [reply docs/development/achievements.md:5]: Good spot! That changed recently when we scoped everything per theme. Feel free to edit.

**Review by FostUK — APPROVED** [review]
Looks great! 🦞

**Review by AnthonyGoodchild — APPROVED** [review]
😎

**Pull request merged**